cms/templates/widgets/footer.html

```diff
@@ -4,7 +4,7 @@
 <div class="wrapper-footer wrapper">
   <footer class="primary" role="contentinfo">
     <div class="colophon">
-      <p>&copy; 2014 <a href="http://www.edx.org" rel="external">edX</a>. ${ _("All rights reserved.")}</p>
+      <p>&copy; ${settings.COPYRIGHT_YEAR} <a href="http://www.edx.org" rel="external">edX</a>. ${ _("All rights reserved.")}</p>
```

Comment:
Actually - can you make this use {platform_name}, too? We're trying to get all branding out of Studio as well as LMS. This should have been done in #6309 but was obviously missed.